### PR TITLE
sioyek: add version 1.1.0

### LINKED ISSUE #8098: [Request] Sioyek
### Information

**Name:** Sioyek

**Description:** Sioyek is a PDF viewer designed for reading research papers and technical books 

**Homepage:** https://sioyek.info/

**Download link(s):** https://github.com/ahrm/sioyek/releases

**Some indication of popularity/repute:** GitHub stars 1.9k

### PR DESCRIPTION
**Pull request by tech189**
- Closes #8098
- Persists user config and last opened file
- Should we suggest vcredist & delete it from `$dir`? It is included in the portable zip and is 24MB.

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

File: bucket/sioyek.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    "homepage": "https://sioyek.info/",
+    "license": "GPL-3.0-only",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/ahrm/sioyek/releases/download/v1.1.0/sioyek-release-windows-portable.zip",
+            "hash": "29c65d4a9932199f4cdf46aca5950518d94dec27aca03e6ebaeee88121bea8f4"
+        }
+    },
+    "extract_dir": "sioyek-release-windows",
+    "pre_install": "New-Item \"$dir\\last_document_path.txt\" | Out-Null",
```
Comment (issaclin32) [inline bucket/sioyek.json:13]:
```suggestion
    "pre_install": [
        "if (!(Test-Path \"$persist_dir\\last_document_path.txt\"))New-Item \"$dir\\last_document_path.txt\" | Out-Null",
        "'test.db', 'local.db', 'shared.db' | ForEach-Object {",
        "    if (Test-Path \"$persist_dir\\$_\") {Copy-Item \"$persist_dir\\$_\" \"$dir\\\"}",
        "}"
    ],
```

Those `.db` files will be created after running the app. Also an empty `.db` file will cause the app to crash, so we need to persist them **manually**.

Related code can be found [here](https://github.com/ahrm/sioyek/blob/cffcdcb8381a432fd0c81c99d4499a50ba3324d2/pdf_viewer/main.cpp#L210)

File: bucket/sioyek.json [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    "shortcuts": [
```
Comment (issaclin32) [inline bucket/sioyek.json:14]:
```suggestion
    "uninstaller": {
        "script": [
            "'test.db', 'local.db', 'shared.db' | ForEach-Object {",
            "    if (Test-Path \"$dir\\$_\") {Copy-Item \"$dir\\$_\" \"$persist_dir\\\"}",
            "}"
        ]
    },
    "shortcuts": [
```

**Comment by issaclin32** [comment]
> Should we suggest vcredist & delete it from $dir?

I think we should do that.

File: bucket/sioyek.json [hunk 3]
```diff
@@ -0,0 +1,53 @@
+{
+    "version": "1.1.0",
+    "description": "A PDF viewer designed for reading research papers and technical books",
+    "homepage": "https://sioyek.info/",
+    "license": "GPL-3.0-only",
+    "suggest": {
+        "Microsoft Visual C++ Redistributables 2019": "extras/vcredist2019"
```
Comment (issaclin32) [inline bucket/sioyek.json:7]:
```suggestion
        "Microsoft Visual C++ Redistributables": "extras/vcredist2022"
```
VCRedist 2022 now [supports all functionalities of VCRedist2015-2022](https://docs.microsoft.com/en-us/cpp/windows/latest-supported-vc-redist?view=msvc-170#visual-studio-2015-2017-2019-and-2022)

Comment (tech189) [reply bucket/sioyek.json:7]: Neat! Thanks

**Review by issaclin32 — APPROVED** [review]

**Pull request merged**